app.js

```diff
@@ -8,42 +8,114 @@ app.use(logger("dev"));
 app.use(express.static("public"));
 app.use(express.json());
 
-
 // Iteration 1 - Connect to MongoDB
 // DATABASE CONNECTION
 
+const mongoose = require("mongoose");
+
+const MONGODB_URI = "mongodb://127.0.0.1:27017/express-mongoose-recipes-dev";
 
+mongoose
+  .connect(MONGODB_URI)
+  .then((res) => {
+    console.log(`connected to database: ${res.connections[0].name}`);
+  })
+  .catch(() => {
+    console.error("Error connecting to database", err);
+  });
 
 // ROUTES
-//  GET  / route
-app.get('/', (req, res) => {
-    res.send("<h1>Lab | Express Mongoose Recipes</h1>");
+//  GET  / route - This is just an example route
+app.get("/", (req, res) => {
+  res.send("<h1>LAB | Express Mongoose Recipes</h1>");
 });
 
-
-//  Iteration 4 - Create recipe route
+//  Iteration 3 - Create a Recipe route
 //  POST  /recipes route
 
+const Recipe = require("./models/Recipe.model");
+
+app.post("/recipes", (req, res) => {
+  Recipe.create({
+    title: req.body.title,
+    instructions: req.body.instructions,
+    level: req.body.level,
+    ingredients: req.body.ingredients,
+    image: req.body.image,
+    duration: req.body.duration,
+    isArchived: req.body.isArchived,
+    created: req.body.created,
+  })
```

Comment:
Also something that we can do is destructuring the props from the body something like this
const { title, instructions, level, ingredients, image ,duration, isArchived, created} = req.body
```suggestion
app.post("/recipes", (req, res) => {
  Recipe.create({
    title,
    instructions,
    level,
    ingredients,
    image,
    duration,
    isArchived,
    created,
  })
```